utils/format/balance.ts

```diff
@@ -10,3 +10,25 @@ export const formatBalanceEmptyOnZero = (
     ? ''
     : trimAll(formatBalance(amount, decimals || 12, symbol || 'KSM'))
 }
+
+export const formatBsxBalanceEmptyOnZero = (
+  amount: string,
+  decimals?: number,
+  symbol?: string
+) => {
```

Comment:
adding return type here?
```suggestion
): string => {
```